### PR TITLE
Extract MAX_FINALITY_LAG constant from relay_chain_selection

### PR DESCRIPTION
**Pull request by tifecool**
PR attempts to solve #4788 

Used MAX_HEADS_LOOK_BACK in approval voting as constant that MAX_BATCH_SCRAPE_ANCESTORS in dispute coordinator and MAX_FINALITY_LAG in relay chain selection are bound to.

polkadot address: 1s51dYXziQwTnSnModErKfra3TeEsVipqzY846CLCTEXtHJ

### CODE REVIEW

File: node/core/dispute-coordinator/Cargo.toml [hunk 1]
```diff
@@ -17,6 +17,7 @@ polkadot-primitives = { path = "../../../primitives" }
 polkadot-node-primitives = { path = "../../primitives" }
 polkadot-node-subsystem = { path = "../../subsystem" }
 polkadot-node-subsystem-util = { path = "../../subsystem-util" }
+polkadot-node-core-approval-voting = { path = "../approval-voting" }
```
Comment (rphmeier) [inline node/core/dispute-coordinator/Cargo.toml:20]: We usually avoid depending on subsystems from one another. I'd recommend putting the constant into `polkadot-node-primitives` and having all of them draw it out from there.

**Comment by tifecool** [comment]
Moved constant to node primitive 👌

File: node/primitives/src/lib.rs [hunk 2]
```diff
@@ -76,6 +76,11 @@ pub const BACKING_EXECUTION_TIMEOUT: Duration = Duration::from_secs(2);
 /// dispute participants.
 pub const APPROVAL_EXECUTION_TIMEOUT: Duration = Duration::from_secs(6);
 
+/// Linked to MAX_FINALITY_LAG in relay chain selection,
+/// MAX_HEADS_LOOK_BACK in approval voting and
+/// MAX_BATCH_SCRAPE_ANCESTORS in dispute coordinator
```
Comment (ordian) [inline node/primitives/src/lib.rs:81]:
```suggestion
/// Linked to `MAX_FINALITY_LAG` in relay chain selection,
/// `MAX_HEADS_LOOK_BACK` in `approval-voting` and
/// `MAX_BATCH_SCRAPE_ANCESTORS` in `dispute-coordinator`
```

**Review by ordian — APPROVED** [review]

**Comment by ordian** [comment]
`cargo +nightly fmt` is failing

File: node/core/dispute-coordinator/src/real/initialized.rs [hunk 3]
```diff
@@ -70,8 +70,8 @@ use super::{
 
 // The capacity and scrape depth are equal to the maximum allowed unfinalized depth.
 const LRU_SCRAPED_BLOCKS_CAPACITY: usize = 500;
```
Comment (ordian) [inline node/core/dispute-coordinator/src/real/initialized.rs:72]:
```suggestion
const LRU_SCRAPED_BLOCKS_CAPACITY: usize = MAX_FINALITY_LAG as usize;
```

**Pull request merged**